omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderFrequencyResource1_10.java

```diff
@@ -68,6 +69,30 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	/**
+	 * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getCreatableProperties()
+	 */
+	@Override
+	public DelegatingResourceDescription getCreatableProperties() {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		
+		description.addRequiredProperty("frequencyPerDay");
```

Comment:
How about the name?

Reply:
Name was got from [here](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/OrderFrequency.java#L25).

Reply:
What does that mean?

Reply:
It means order frequency for each day.

Reply:
What does "Name was got from here" mean?

Reply:
I am thinking, you are inquiring how I came about with the name "`frequencyPerDay`". "[here](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/OrderFrequency.java#L25)" is the link to what made me choose this name. Unless I did not get you well or my communication skill poor, you may throw more light/meaning to the "name" in question, or what I'm missing.

Reply:
I think i confused you. Let me put it differently. How about the name property?

Reply:
Oh yes, the name. I intentionally overrode  `getCreatableProperties()` not to include the super class's name and description properties because when doing a GET, these properties are filled using the `[concept](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/OrderFrequency.java#L105-L120)` 's name and description. If I included them, they would be redundant or even misleading when the names and descriptions differ while POSTing OrderFrequency against the referenced Concept.